cas/incident-reporting.md

```diff
@@ -0,0 +1,101 @@
+# Incidents
+
+Incidents can happen. Things do not always go as planned, and that can be okay. However, when incidents occur, the underlying issues (i.e. root causes) must be identified and remediated to discourage the incident from occurring again. Formally documenting the incident in a report encourages an understanding of all contributing root causes, and it presents the opportunity to clearly communicate a remediation plan to reduce the probability of recurrence. 
+
+## Objectives and Expectations
+
+The purpose of Incident Reporting is to help us work together to build a more secure web. 
+
+Depending on the root programs in which a CA Owner participates, it may be required to:
+* create [Audit Incident Reports](audit-incident-report) when audits have non-conformities, qualifications, or modified opinions; and/or
+* create [Incident Reports](incident-report) for all other incidents.
+
+These reports provide lessons learned and transparency about the steps the CA Owner takes to address the immediate issue and prevent future issues. If the underlying problem goes unfixed, then other issues that share the same root cause will subsequently surface. Additionally, Incident Reports help the Web PKI ecosystem as a whole because they promote continuous improvement, information sharing, and highlight opportunities to define and adopt improved practices, policies, and controls.
+
+CA Incident Reports are expected to:
+- be candid, timely, and transparent;
+- clearly and objectively communicate the scope, impact, and severity of an incident;
+- help community members understand the relevant architectures, implementations, operations, and external dependencies surrounding an incident;
+- demonstrate a detailed understanding of the root cause(s) of an incident;
+- unambiguously explain how systems, processes, and/or policies failed; 
+- describe the circumstances that prevented the problem(s) from being detected earlier;
+- describe a clear timeline of a CA Owner's actions while responding to and remediating an incident;
+- include a detailed and measurable explanation of actions taken or planned by the CA Owner that demonstrate a substantive commitment of how the CA Owner's systems, policies, and/or processes will be made more robust (i.e., demonstrate continuous improvement); and
+- share lessons learned that could be helpful to all CA Owners in building better systems policies, and/or processes.
+
+### Incident Reporting: Good Practices
```

Comment:
...but maybe that's best done by merging the expectations with the good practices: a list of expectations and the practices that we recommend for meeting those expectations.